### DISCUSSION TITLE
Is it possible to make a transparent window in software rendering mode?

### DISCUSSION THREAD

**Discussion by dme-compunet** [AvaloniaUI/Avalonia#13913] [Q&A] [answered]
I have an app with semi-transparent windows, unfortunately the transparency is only preserved in GPU rendering mode, but when I work in software rendering mode the transparency disappears and a white background appears instead.

Is there anything to do?

**Comment by timunie** [comment]
> Is there anything to do?

tell us:
- your Avalonia version
- OS
- Steps to reproduce

**Reply by dme-compunet** [reply]
I'm on Windows 11 and Avlonia 11.0.6,

Steps to reproduce:
1) Create a new project
2) Add this code to the MainWindow constructor :
```csharp
Background = null;
TransparencyLevelHint = [WindowTransparencyLevel.Transparent];

Content = new Ellipse
{
    Fill = Brushes.Red
};
```
3) Run, and see that the window is transparent
4) Go to software rendering mode with:
```csharp
.With(new Win32PlatformOptions
{
    RenderingMode = [Win32RenderingMode.Software]
})
```
5) Run and see that the window has lost its transparency

Same with popups.

Is it better to open an Issue about it? (I thought it was something familiar)

**Reply by timunie** [reply]
I'm not sure if that is even supported for software rendering. @kekekeks what do you think, should this be an issue or is it by design on Windows?

**Reply by timunie** [reply, marked as answer]
@dme-compunet can you check with nightly? I cannot reproduce it using latest nightly version on my end

**Comment by dme-compunet** [comment]
I can confirm that the nightly build does not have this issue,
when do we expect to see this fix in a stable version? Is this something
that was recently fixed?

**Reply by timunie** [reply]
probably. Can be there in 11.0.7 (if the fix was able to be backported) or latest in 11.1. I've no due date yet for you.

**Comment by dme-compunet** [comment]
Thanks for your help!

**Comment by SmRiley** [comment]
This issue is fixed in 11.1.0-rc1

**Reply by SmRiley** [reply]
But there is still a white background fill flickering at startup
